packages/driver/src/cypress/error_messages.coffee

```diff
@@ -615,15 +615,15 @@ module.exports = {
       Before the page load, you were bound to the origin policy:
         > #{originPolicy}
 
-      A cross origin error happens when your application navigates to a new superdomain which does not match the origin policy above.
+      A cross origin error happens when your application navigates to a new domain which does not match the origin policy above.
 
       This typically happens in one of three ways:
 
       1. You clicked an <a> that routed you outside of your application
       2. You submitted a form and your server redirected you outside of your application
       3. You used a javascript redirect to a page outside of your application
 
-      Cypress does not allow you to change superdomains within a single test.
+      Cypress does not allow you to navigate to different origin domain within a single test.
```

Comment:
```suggestion
      Cypress does not allow you to navigate to different origin within a single test.
```

And then, maybe below this, define what an "origin" is (scheme + host + port?)

Reply:
It's complicated, which is why I just link out do the docs url. Because it's really you can't navigate to another URL that has a different `host` (unless of the same superdomain), `protocol`, or `port`. 😬 Does that seem clear enough of a description?

https://docs.cypress.io/guides/guides/web-security.html#Limitations

Reply:
Yeah, I think that's fine, it's enough to give them a good hint at least.